src/celestia/audiosession.h

```diff
@@ -0,0 +1,62 @@
+#pragma once
+
+#include <celcompat/filesystem.h>
+
+namespace celestia
+{
+
+namespace constants
```

Comment:
`constants` is too generic name. either add all audio code into `celestia::audio` or remove `namespace constants`

Reply:
removed